e2e/nextjs/next.config.mjs

```diff
@@ -3,7 +3,6 @@ import { withHighlightConfig } from '@highlight-run/next/config'
 
 const nextConfig = {
 	experimental: {
-		appDir: true,
```

Comment:
don't we use app dir in the e2e next example? should we keep this true?

Reply:
This is no longer necessary in Next 14.